### PR TITLE
add Santiago's answers

### PR DESCRIPTION
**Pull request by smola**

### CODE REVIEW

File: talent/sourcerers/smola.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+**General**
+- What's your name? How do your friends call you? 
+  - Santiago Moisés, Santi
+- When were you born?
+  - 1987
+- Where are you from? Where have you lived?
+  - [Alicante](https://en.wikipedia.org/wiki/Alicante) → [Valencia](https://en.wikipedia.org/wiki/Valencia) → [Madrid](https://en.wikipedia.org/wiki/Madrid) ([Spain](https://en.wikipedia.org/wiki/Spain))
+- What languages (natural and programming ones) do you know?
+  - Spanish, English, Catalonian, Go, Java, Scala, Python and a few others I am not proficient with or I would prefer to forget.
```
Comment (campoy) [inline talent/sourcerers/smola.md]: The language is Catalan

File: talent/sourcerers/smola.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- What are your hobbies and what do you like about them?
+  - Most computer-related stuff.
+- What was the last lie you told?
+  - Something in this questionnaire.
+- What was your first thought when you woke up this morning?
+  - $@!# what happened to the alarm?
+- What question do you hate to answer?
+  - *Tell me three of your virtues and three defects.*
+- What’s your favorite knock-knock joke?
+  - Not a common thing in Spain as far as I know.
```
Comment (campoy) [inline talent/sourcerers/smola.md]: Maybe we should change it to: what's your favorite "camarero!" song?

Comment (smola) [reply talent/sourcerers/smola.md]: Or maybe something that has a lower cultural bias.

Comment (smola) [reply talent/sourcerers/smola.md]: It seems that no spaniard could answer the question? On the other hand, the way @gomesfernanda [adapted the answer](https://github.com/src-d/guide/blob/master/talent/sourcerers/gomesfernanda.md] was hilarious. Although I guess it works only in Portuguese and Spanish.

Comment (campoy) [reply talent/sourcerers/smola.md]: > Hi, I'm Tunisian, and I'm here to kill you!
> To what?
> Tunisian

**Pull request merged**